samanthas_telegram_bot/data_structures/context_types.py

```diff
@@ -8,103 +8,70 @@
 
 from samanthas_telegram_bot.api_clients.auxil.constants import DataDict
 from samanthas_telegram_bot.conversation.auxil.enums import ConversationMode
-from samanthas_telegram_bot.conversation.auxil.load_phrases import load_phrases
 from samanthas_telegram_bot.data_structures.constants import Locale
 from samanthas_telegram_bot.data_structures.enums import AgeRangeType, Role
 from samanthas_telegram_bot.data_structures.models import (
+    AgeRange,
     Assessment,
     AssessmentAnswer,
     DayAndTimeSlot,
     LanguageAndLevel,
+    MultilingualBotPhrase,
     SmalltalkResult,
     TeacherPeerHelp,
 )
 
 
+@dataclass
 class BotData:
```

Comment:
`BotData` is now a dataclass, and almost all attributes (but one) are here for type hinting and docstrings, as well as, obviously, to avoid creating attributes after initialization. The actual data is written into these attributes in `application_start/bot_data_loader`.